app/cosi/README.md

```diff
@@ -0,0 +1,338 @@
+[![Build Status](https://travis-ci.org/dedis/cothority.svg?branch=master)](https://travis-ci.org/dedis/cothority)
```

Comment:
Or having the wiki point to the README for specific instructions? Anyway, that's probably also a later issue.